### PR TITLE
My app, in mithril + spectre.css

### PR DESCRIPTION
**Pull request by russitto**
Hi, this is my version.
Code at https://gitlab.com/russitto/hnsm
App: https://hnsm-66e89.firebaseapp.com/#!/

:)

### CODE REVIEW

**Comment by WebReflection** [comment]
out of curiosity, it looks like your app would never update 'till cached?
https://gitlab.com/russitto/hnsm/blob/master/source/sw.js#L45-48

**Comment by russitto** [comment]
@WebReflection its ok your comment, but service worker fetch event is not working for me anyways :(
This lines never execute.

**Comment by housseindjirdeh** [comment]
@russitto Thank you for submitting this, and apologies on my part for being off the ball on this project for some time. We're in the process of clearing/approving all submissions at the moment now so please let us know if you're still interested in exploring improving this app so we can land it :)

Running some newer traces and it looks like your application is showing consistently acceptable TTI times 🚀

Regular 3G: https://www.webpagetest.org/lighthouse.php?test=180701_P9_86447aa8432cc002c49f720c678ffb49&run=2

Slow 3G: https://www.webpagetest.org/lighthouse.php?test=180701_PQ_2b8cd6240249dd9266d9bc71a8ee0c3f&run=2

Looks like your SW is also registering successfully, but it doesn't seem to serve any of the assets when I view the webpage for a second time offline 🤔 Do you happen to know why this is by any chance?

Another minor thing (design related), but comment threads can be a little hard to parse through on mobile without much spacing with each other.

screen shot 2018-06-30 at 9 14 50 pm
screen shot 2018-06-30 at 9 16 16 pm

Feel free to disregard this as it isn't a hard requirement, but will be nice if you can add a little padding between comments (and maybe bold/enlarge the details for each comment [user and time posted]).

And lastly, you're working with the same API that I used when I built a client some time ago, but there's a newer CDN cached API that can probaby serve your results faster. Please explore using it if you can! Link here: https://github.com/davideast/hnpwa-api

I'm going to close this for now since we've had it open for a very long time. But please let us know if you're interested in working on some of these minor changes and we'll reopen before landing it in :)
